docs/docs/index.md

```diff
@@ -13,19 +13,20 @@ The framework supports the following types of agents:
 
 Key features of Koog include:
 
-- **Pure Kotlin implementation**: Build AI agents entirely in natural and idiomatic Kotlin.
-- **MCP integration**: Connect to Model Control Protocol for enhanced model management.
-- **Embedding capabilities**: Use vector embeddings for semantic search and knowledge retrieval.
-- **Custom tool creation**: Extend your agents with tools that access external systems and APIs.
-- **Ready-to-use components**: Speed up development with pre-built solutions for common AI engineering challenges.
-- **Intelligent history compression**: Optimize token usage while maintaining conversation context using various pre-built strategies.
+- **Multiplatform development**: Deploy agents across JVM, JS, WasmJS, Android, and iOS targets using Kotlin Multiplatform.
+- **Reliability and fault-tolerance**: Handle failures with built-in retries and restore full agent state machines with the agent persistence feature.
```

Comment:
> restore full agent state machines

Unfortunately, it's not quite at this stage yet. Can we put it more mildly for now?

Reply:
Yes, this sounds better